vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/view/ProjectViewPage.java

```diff
@@ -27,37 +30,63 @@ public class ProjectViewPage extends Div implements
-      ProjectLinksComponent projectLinksComponent) {
-    handler = new ProjectViewHandler(projectDetailsComponent, projectLinksComponent);
+  public ProjectViewPage(@Autowired ProjectDetailsComponent projectDetailsComponent,
+      @Autowired ProjectLinksComponent projectLinksComponent,
+      @Autowired ExperimentalDesignDetailComponent experimentalDesignDetailComponent) {
+    handler = new ProjectViewHandler(projectDetailsComponent,
+        projectLinksComponent, experimentalDesignDetailComponent);
     add(projectDetailsComponent);
     add(projectLinksComponent);
+    //ToDo Replace with Dedicated Navbar component and routing
+    initNavbar(projectDetailsComponent, experimentalDesignDetailComponent);
+    add(navBar);
     setPageStyles();
-    setComponentStyles(projectDetailsComponent, projectLinksComponent);
+    setComponentStyles(projectDetailsComponent, projectLinksComponent, experimentalDesignDetailComponent);
     log.debug(
         String.format("New instance for project view (#%s) created with detail component (#%s)",
             System.identityHashCode(this), System.identityHashCode(projectDetailsComponent)));
   }
 
+  private void initNavbar(ProjectDetailsComponent projectDetailsComponent, ExperimentalDesignDetailComponent experimentalDesignDetailComponent) {
+    navBar.add(switchComponentsButton);
+    switchComponentsButton.addClickListener(clickEvent -> switchDetailsComponents(projectDetailsComponent, experimentalDesignDetailComponent));
+  }
+
+  private void switchDetailsComponents(ProjectDetailsComponent projectDetailsComponent, ExperimentalDesignDetailComponent experimentalDesignDetailComponent){
+      if(this.getChildren().toList().contains(projectDetailsComponent)){
+        add(experimentalDesignDetailComponent);
+        remove(projectDetailsComponent);
+      } else if (this.getChildren().toList().contains(experimentalDesignDetailComponent)) {
+        remove(experimentalDesignDetailComponent);
+        add(projectDetailsComponent);
+      }
+  }
+
   @Override
   public void setParameter(BeforeEvent beforeEvent, String s) {
     handler.routeParameter(s);
 
     log.debug("Route '" + ROUTE + "' called with parameter '" + s + "'");
   }
-  public void setPageStyles(){
+
+  public void setPageStyles() {
     addClassNames("project-view-page");
   }
-  public void setComponentStyles(ProjectDetailsComponent projectDetailsComponent, ProjectLinksComponent projectLinksComponent){
+
+  public void setComponentStyles(ProjectDetailsComponent projectDetailsComponent,
+      ProjectLinksComponent projectLinksComponent,
+      ExperimentalDesignDetailComponent experimentalDesignDetailComponent) {
     projectDetailsComponent.setStyles("project-details-component");
     projectLinksComponent.setStyles("project-links-component");
+    //Todo Determine if we want to have seperate styles for each component
+    experimentalDesignDetailComponent.setStyles("experimental-design-component");
```

Comment:
why not `addClassName` in the `ExperimentalDesignDetailComponent` directly?

Reply:
Because the page should decide on how the component should be styled. (In this case it's the decision where in the css grid the component should be shown)